cmd/skaffold/app/cmd/cmd.go

```diff
@@ -310,3 +308,28 @@ func preReleaseVersion(s string) bool {
 	}
 	return false
 }
+
+func isQuiet() (string, bool) {
+	switch {
+	case !interactive:
+		return "Update check prompt, survey prompt and telemetry disabled in non-interactive mode", true
+	case quietFlag:
+		return "Update check prompt, survey prompt and telemetry disabled in quiet mode", true
+	case analyze:
+		return "Update check prompt, survey prompt and telemetry disabled disabled when running `init --analyze`", true
+	default:
+		return "", false
+	}
+}
+
+func updateCheckForReleasedVersionsIfNotDisabled(s string) string {
```

Comment:
nit: does this name need to be so long? 😄  `updateCheckForVersion()` seems like enough to me

Reply:
I wanted it actually convey what the function did because it has been so confusing in the past :)